.github/workflows/code-style-lint.yml

```diff
@@ -32,5 +32,5 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: |
-          cd source && npm i --only=dev
+          cd source && npm i
```

Comment:
should this be doing a dev install like the previous line? If so can you still do `production=false`?

Reply:
It seems reasonable to run the linter on all source and not just prod deployment files if we can